### PR TITLE
Remove dependency of OCR secret from the Verrazzano installation

### PR DESCRIPTION
**Pull request by mgianatagh**
Remove the dependency of create a secret named `ocr` from the verrazzano installation.  A secret of that name is only required if you install one of the sample applications.  The sample includes separate instructions for creating dependent secrets.

### CODE REVIEW

File: install/README.md [hunk 1]
```diff
@@ -37,17 +37,7 @@ The following software must be installed on your system.
...
-
-```
-   kubectl create secret docker-registry ocr \
-       --docker-username=<username> \
-       --docker-password=<password> \
-       --docker-server=container-registry.oracle.com
-```
-
-### 3. Do the install
+### 2. Do the install
```
Comment (bobdonat) [inline install/README.md:40]: Do steps need to be renumbered after this step since the original step 2 has been removed.

Comment (mgianatagh) [reply install/README.md:40]: Yes, not sure how I missed those.

Comment (mgianatagh) [reply install/README.md:40]: I pushed the fix

**Review by markxnelson — APPROVED** [review]
LGTM

**Review by bobdonat — APPROVED** [review]

**Pull request merged**